x-pack/plugins/security_solution/cypress/integration/detection_rules/bulk_edit_rules.spec.ts

```diff
@@ -109,15 +113,21 @@ describe('Detection rules, bulk edit', () => {
     // check modal window for few selected rules
     selectNumberOfRules(5);
     clickAddIndexPatternsMenuItem();
-    waitForElasticRulesBulkEditModal(5);
+    checkElasticRulesCannotBeModified(5);
     cy.get(MODAL_CONFIRMATION_BTN).click();
 
-    // Select Elastic rules and custom rules, check mixed rules warning modal window, proceed with editing custom rules
+    // Select all rules(Elastic rules and custom)
     cy.get(ELASTIC_RULES_BTN).click();
     selectAllRules();
     clickAddIndexPatternsMenuItem();
-    waitForMixedRulesBulkEditModal(totalNumberOfPrebuiltRules, 6);
-    cy.get(MODAL_CONFIRMATION_BTN).should('have.text', 'Edit custom rules').click();
+    waitForMixedRulesBulkEditModal(6);
+
+    // check rules that cannot be edited for index patterns: immutable and ML
+    checkElasticRulesCannotBeModified(totalNumberOfPrebuiltRules);
+    checkMachineLearningRulesCannotBeModified(1);
```

Comment:
It used only once and within a single test, I'm not sure this can be more readable

```typescript
const MACHINE_LEARNING_RULES_COUNT = 1;
...
checkMachineLearningRulesCannotBeModified(MACHINE_LEARNING_RULES_COUNT);
```

Reply:
values moved to variables